packages/tasks/src/model-libraries.ts

```diff
@@ -1013,6 +1013,22 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		snippets: snippets.sam2,
 		countDownloads: `path_extension:"pt"`,
 	},
+	"sam-3d-body": {
+		prettyLabel: "SAM 3D Body",
+		repoName: "SAM 3D Body",
+		repoUrl: "https://github.com/facebookresearch/sam-3d-body",
+		filter: false,
+		snippets: snippets.sam_3d_body,
+		countDownloads: `path:"model_config.yaml"`,
+	},
+	"sam-3d-object": {
+		prettyLabel: "SAM 3D Object",
+		repoName: "SAM 3D Object",
```

Comment:
I've just realized that https://huggingface.co/facebook/sam-3d-objects/discussions/3 introduced the tag `sam-3d-objects` (with an "s"). The Github repo is also named [sam-3d-objects](https://github.com/facebookresearch/sam-3d-objects) so I'm leaning towards harmonizing to a plural form.

@merveenoyan ok for you?


```suggestion
	"sam-3d-objects": {
		prettyLabel: "SAM 3D Objects",
		repoName: "SAM 3D Objects",
```